### PR TITLE
add: uwufetch

### PR DESCRIPTION
**Pull request by lexisother**

### CODE REVIEW

File: anda/others/uwufetch/anda.hcl [hunk 1]
```diff
@@ -0,0 +1,6 @@
+project pkg {
+	arches = ["x86_64"]
```
Comment (lleyton) [inline anda/others/uwufetch/anda.hcl]: Is this really arch dependent?

File: anda/others/uwufetch/uwufetch.spec [hunk 2]
```diff
@@ -0,0 +1,36 @@
+%define debug_package %nil
+
+Name:			uwufetch
+Version:		2.1
+Release:		1%?dist
+Summary:		A meme system info tool for Linux, based on nyan/uwu trend on r/linuxmasterrace. 
+License:		GPL-3.0
+URL:			https://github.com/TheDarkBug/uwufetch
+BuildRequires:	make gcc git anda-srpm-macros
```
Comment (lleyton) [inline anda/others/uwufetch/uwufetch.spec]: Format.

File: anda/others/uwufetch/uwufetch.spec [hunk 3]
```diff
@@ -0,0 +1,36 @@
+%define debug_package %nil
```
Comment (lleyton) [inline anda/others/uwufetch/uwufetch.spec]: Why is the debug package disabled?

File: anda/others/uwufetch/uwufetch.spec [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+make install DESTDIR=%{?buildroot}/usr
+mkdir %{?buildroot}/usr/lib64
+mv %{?buildroot}/usr/lib/libfetch.so %{?buildroot}/usr/lib64/libfetch.so
+rm -rf %{?buildroot}/usr/include
+
+%files
+/usr/lib/uwufetch/*
+%_libdir/libfetch.so
+/usr/share/man/man1/uwufetch.1.gz
+/usr/bin/uwufetch
```
Comment (lleyton) [inline anda/others/uwufetch/uwufetch.spec]: Try to use macros here.

File: anda/others/uwufetch/uwufetch.spec [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+%install
+make install DESTDIR=%{?buildroot}/usr
+mkdir %{?buildroot}/usr/lib64
+mv %{?buildroot}/usr/lib/libfetch.so %{?buildroot}/usr/lib64/libfetch.so
+rm -rf %{?buildroot}/usr/include
+
+%files
+/usr/lib/uwufetch/*
+%_libdir/libfetch.so
```
Comment (lleyton) [inline anda/others/uwufetch/uwufetch.spec]: Maybe we should split the .so file into a libfetch(-devel) package? What do you think @madonuko?

**Review by lleyton — DISMISSED** [review]

**Review by madonuko — APPROVED** [review]

**Review by korewaChino — APPROVED** [review]

**Pull request merged**